R/descriptives.R

```diff
@@ -1634,12 +1634,22 @@ DescriptivesInternal <- function(jaspResults, dataset, options) {
 
 .descriptivesSdCI <- function(data, options, jaspResults, variableName) {
   ciWidth <- options[["sdCiLevel"]]
-  stateContainerName <- paste0("bootstrapSamples", variableName)
-  sds <- .bootstrapStats(data, options, jaspResults, stateContainerName)$sds
-  percentiles <- (1 + c(-ciWidth, ciWidth)) / 2
-  CIs <- quantile(sds, probs = percentiles)
-  lowerBound <- CIs[1]
-  upperBound <- CIs[2]
+  if (options[["sdCiMethod"]] == "bootstrap") {
+    stateContainerName <- paste0("bootstrapSamples", variableName)
+    sds <- .bootstrapStats(data, options, jaspResults, stateContainerName)$sds
+    percentiles <- (1 + c(-ciWidth, ciWidth)) / 2
+    CIs <- quantile(sds, probs = percentiles)
+    lowerBound <- CIs[1]
+    upperBound <- CIs[2]
+  } else if (options[["sdCiMethod"]] == "chiSquaredModel") {
+    sigma <- sd(data)
+    n <- length(data)
+    alpha <- 1 - ciWidth
+    chiSqLower <- qchisq(alpha / 2, df = n - 1)
+    chiSqUpper <- qchisq(1 - alpha / 2, df = n - 1)
+    lowerBound <- sqrt((n - 1) * sigma^2 / chiSqUpper)
+    upperBound <- sqrt((n - 1) * sigma^2 / chiSqLower)
```

Comment:
```suggestion
    CIs <- stats::quantile(sds, probs = percentiles)
    lowerBound <- CIs[1]
    upperBound <- CIs[2]
  } else if (options[["sdCiMethod"]] == "chiSquaredModel") {
    sigma <- stats::sd(data)
    n <- length(data)
    alpha <- 1 - ciWidth
    chiSqLower <- stats::qchisq(alpha / 2, df = n - 1)
    chiSqUpper <- stats::qchisq(1 - alpha / 2, df = n - 1)
    lowerBound <- sqrt((n - 1) * sigma^2 / chiSqUpper)
    upperBound <- sqrt((n - 1) * sigma^2 / chiSqLower)
```
a (Wikipedia) reference to the equations as a comment would also be nice.